ml/generators/mip_generator.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, dims=(120, 120, 1), batch_size=16,
 
         # Get npy files from Google Cloud Storage
         gcs_client = storage.Client.from_service_account_json(
-            'credentials/client_secret.json'
+            '../credentials/client_secret.json'
```

Comment:
I don't think this is correct. All files should be called from the root, where the credentials folder is.